recipes/full_finetune_single_device.py

```diff
@@ -212,6 +212,7 @@ def setup(self, cfg: DictConfig) -> None:
             enable_activation_checkpointing=cfg.enable_activation_checkpointing,
             compile_model=self._compile,
             model_state_dict=ckpt_dict[training.MODEL_KEY],
+            quantizer_cfg=cfg.get("quantizer", None),
```

Comment:
This seems to follow the same pattern as QAT. I think it makes sense, unless we have reason to worry that the user may be confused about it.

```
# QAT arguments
quantizer:
  _component_: torchtune.training.quantization.Int8DynActInt4WeightQATQuantizer
  groupsize: 256
```

I think that we should add a "enabled" flag, like we use for profiler. That way, we can add it to every config, but have enabled=False as default (or True, if we have torchao as a dependency again at some point).

Then, you would have to do:

```if quantizer_cfg is not None and quantizer_cfg.enabled:
	quantizer = config.instantiate(quantizer_cfg)
	model = quantizer.prepare(model)```